TwoPage/src/main/java/com/microsoft/device/display/samples/twopage/extensions/ScreenInfo.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ *
+ */
+
+package com.microsoft.device.display.samples.twopage.extensions
+
+import android.view.Surface
+import com.microsoft.device.dualscreen.ScreenInfo
+
+/**
+ * Returns [true] if the screen is in portrait mode, [false] otherwise
+ */
+val ScreenInfo.isInPortrait: Boolean
+    get() = getScreenRotation() == Surface.ROTATION_90 ||
+        getScreenRotation() == Surface.ROTATION_270
```

Comment:
Github highlights that there is not a new line after this, probably good to add it. Even though we have that disabled with ktlint, here looks better with a new line.

Reply:
Done